pkg/lightning/config/config.go

```diff
@@ -762,54 +765,6 @@ func (cfg *Config) CheckAndAdjustTiDBPort(ctx context.Context, mustHaveInternalC
 	return nil
 }
 
-func (cfg *Config) CheckAndAdjustFilePath() error {
```

Comment:
this check-and-adjust does several things:

1. if the data-source-dir is a path, we `common.IsDirExists` it. this is going to be replaced by the ListObjects permission check.
2. turn the path into a URL. this comes for free by `storage.ParseBackend`.
3. ban unsupported schemes. currently all schemes are supported (including GCS), so the check is useless now.